### PR TITLE
Merging template updates 2.9

### PR DESCRIPTION
**Pull request by d4straub**
This is merging nf-core template updates 2.9 https://github.com/nf-core/ampliseq/pull/607

I kept TODO strings where not solved yet for separate PRs:
- consider using nf-validation plugin
- MultiQC's method section

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -658,19 +657,36 @@
...
+                    "hidden": true,
+                    "help_text": "By default, parameters set as _hidden_ in the schema are not shown on the command line when a user runs with `--help`. Specifying this option will tell the pipeline to show all parameters."
+                },
+                "validationFailUnrecognisedParams": {
+                    "type": "boolean",
+                    "fa_icon": "far fa-check-circle",
+                    "description": "Validation of parameters fails when an unrecognised parameter is found.",
+                    "default": false,
+                    "hidden": true,
+                    "help_text": "By default, when an unrecognised parameter is found, it returns a warninig."
```
Comment (erikrikarddaniel) [inline nextflow_schema.json:681]:
```suggestion
                    "help_text": "By default, when an unrecognised parameter is found, it returns a warning."
```

**Review by erikrikarddaniel — APPROVED** [review]
I certainly do not have a lot of template experience, but as far as I can see all fine except two tiny things.

File: assets/methods_description_template.yml [hunk 2]
```diff
@@ -3,17 +3,21 @@ description: "Suggested text and references to use when describing pipeline usag
...
+  <p>Data was processed using nf-core/ampliseq v${workflow.manifest.version} ${doi_text} of the nf-core collection of workflows (<a href="https://doi.org/10.1038/s41587-020-0439-x">Ewels <em>et al.</em>, 2020</a>), utilising reproducible software environments from the Bioconda (<a href="https://doi.org/10.1038/s41592-018-0046-7">Grüning <em>et al.</em>, 2018</a>) and Biocontainers (<a href="https://doi.org/10.1093/bioinformatics/btx192">da Veiga Leprevost <em>et al.</em>, 2017</a>) projects.</p>
   <p>The pipeline was executed with Nextflow v${workflow.nextflow.version} (<a href="https://doi.org/10.1038/nbt.3820">Di Tommaso <em>et al.</em>, 2017</a>) with the following command:</p>
   <pre><code>${workflow.commandLine}</code></pre>
+  <p>${tool_citations}</p>
   <h4>References</h4>
   <ul>
-    <li>Straub D, Blackwell N, Langarica-Fuentes A, Peltzer A, Nahnsen S, Kleindienst S. Interpretations of Environmental Microbial Community Studies Are Biased by the Selected 16S rRNA (Gene) Amplicon Sequencing Pipeline. Front Microbiol. 2020 Oct 23;11:550420. <a href="https://doi.org/10.3389/fmicb.2020.550420">https://doi.org/10.3389/fmicb.2020.550420</a></li>
```
[the remark this answers is not in the document]
Comment (d4straub) [reply assets/methods_description_template.yml:14]: That is supposed be regained by `${tool_citations}` in line 13 and `${tool_bibliography}` in line 20 of this file here. That two variables are fed by `toolCitationText` and `toolBibliographyText` in `lib/WorkflowAmpliseq.groovy`, if I understand right. That is still marked by TODO originating from the template.

File: nextflow_schema.json (hunk 1 above)
Comment (d4straub) [reply nextflow_schema.json:681]: Indeed, thanks, good catch!

**Pull request merged**